anko/library/generated/sdk25/src/Properties.kt

```diff
@@ -122,3 +122,12 @@ var android.widget.Toolbar.titleResource: Int
     @Deprecated(AnkoInternals.NO_GETTER, level = DeprecationLevel.ERROR) get() = AnkoInternals.noGetter()
     set(v) = setTitle(v)
 
+var android.widget.TextView.textAppearance: Int
+	@Deprecated(AnkoInternals.NO_GETTER, level = DeprecationLevel.ERROR) get() = AnkoInternals.noGetter()
```

Comment:
Really? About which branch are you talking about? I've checked out master, and it doesn't seem to have one: https://github.com/Kotlin/anko/blob/master/anko/library/generated/sdk23/src/Properties.kt
Btw, what's the purpose of these packages ending with sdk number?